tests/Js/WindowTest.php

```diff
@@ -14,19 +14,21 @@ public function testWindow()
         $webAssert = $this->getAssertSession();
 
         $page->clickLink('Popup #1');
+        $popup1WindowHandle = array_slice($session->getDriver()->getWindowNames(), -1)[0];
         $session->switchToWindow(null);
 
         $page->clickLink('Popup #2');
+        $popup2WindowHandle = array_slice($session->getDriver()->getWindowNames(), -1)[0];
         $session->switchToWindow(null);
 
         $el = $webAssert->elementExists('css', '#text');
         $this->assertSame('Main window div text', $el->getText());
 
-        $session->switchToWindow('popup_1');
+        $session->switchToWindow($popup1WindowHandle);
```

Comment:
since Selenium 3.x and newer Firefox names are handles

and only `handle` must be passed, no `name` is supported (/w or /wo `handle`) - fixed in https://github.com/instaclick/php-webdriver/pull/119

Reply:
Should we preserve backwards-compatible behavior by using the window name on Selenium2 and handle on Selenium3?

Reply:
The ability to switch a window by name on Selenium 3 was fixed in the https://github.com/minkphp/MinkSelenium2Driver/pull/384.